### DISCUSSION TITLE
Navigating Nelle Home

### DISCUSSION THREAD

**Discussion by GitTodojetaime** [sandbox-bio/sandbox.bio#110] [Q&A] [answered]
Starting from /Users/nelle/data, which of the following commands could Nelle use to navigate to her home directory, which is /Users/nelle?

I chose..

Moving up the directory ..cd
Going home ~cd
or..
cd/home/nelle

I am being told these are incorrect. Please let me know if I'm overlooking something!

**Comment by robertaboukhalil** [comment, marked as answer]
If you're at folder `/Users/nelle/data`, getting to `/Users/nelle` would work if you did `cd ..`, `cd ~`, `cd ~/data/..` or even just `cd`

**Reply by Neha-Ashfaq** [reply]
thank you, this helped me too!
